src/core/solution.jl

```diff
@@ -1,18 +1,18 @@
 
 ""
 function solution_fs!(pm::_PM.AbstractPowerModel, sol::Dict{String,Any})
-    sol["fault"] = ref(pm, pm.cnw, :active_fault)
-    sol["fault"]["currents"] = Dict{String, Any}()
-    add_branch_currents!(pm, sol)
-    add_tansformer_currents!(pm, sol)
+    # sol["fault"] = ref(pm, pm.cnw, :active_fault)
+    # sol["fault"]["currents"] = Dict{String, Any}()
+    # add_branch_currents!(pm, sol)
+    # add_tansformer_currents!(pm, sol)
```

Comment:
Is this meant to be disabled?

Reply:
@pseudocubic We cleaned up the solution, we need to develop a format for the output that can be used for protection.